### DISCUSSION TITLE
Welcome to deepbots Discussions!

### DISCUSSION THREAD

**Discussion by tsampazk** [aidudezzz/deepbots#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by eakirtas** [comment]
Yooohoooo
